framework/src/engine/generator/endpoint.ts

```diff
@@ -207,25 +207,25 @@ export class Endpoint {
 		validator.validate<EstimateSafeStatusRequest>(estimateSafeStatusRequestSchema, ctx.params);
 
 		const req = ctx.params;
-
 		const finalizedHeight = this._consensus.finalizedHeight();
-
 		const finalizedBlock = await this._chain.dataAccess.getBlockHeaderByHeight(finalizedHeight);
-		if (finalizedBlock.timestamp < req.timeShutdown) {
+		if (req.timeShutdown > finalizedBlock.timestamp) {
 			throw new Error(`A block at the time shutdown ${req.timeShutdown} must be finalized.`);
 		}
 
-		// assume there is 30 days per month
+		// assume there are 30 days per month
 		const numberOfBlocksPerMonth = Math.ceil((60 * 60 * 24 * 30) / this._blockTime);
-		const heightOneMonthAgo = Math.max(finalizedHeight - numberOfBlocksPerMonth, 0);
+		const heightOneMonthAgo = Math.max(finalizedHeight - numberOfBlocksPerMonth, 1);
```

Comment:
to be precise it should get the genesis block height, but probably + 1 should be fine